integration-tests/kyuubi-jdbc-it/pom.xml

```diff
@@ -96,6 +96,24 @@
             <artifactId>testcontainers-scala-postgresql_${scala.binary.version}</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.dimafeng</groupId>
+            <artifactId>testcontainers-scala-clickhouse_${scala.binary.version}</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>com.clickhouse</groupId>
+            <artifactId>clickhouse-jdbc</artifactId>
```

Comment:
can we use the `http` classifier jar? then we don't need to care about lz4 jar

Reply:
ok,fix it